### ISSUE TITLE
Add acq-<label> to filename of html report

### ISSUE THREAD

**Issue by jhauneuro** [PennLINC/qsiprep#835]
## Summary
I have multiple dwi images in my scan session. When I preprocess these data the sub-X.html report gets overwritten in the output sub-X folder. 

## Additional details
The benefit is to preserve separate html reports for each preprocessed dwi in a scan session.

## Next steps
Adding acq-<label> to the name of the html report could help resolve this issue and prevent overwriting. Adding ses-<label> will also be helpful for datasets with multiple sessions.

**Comment by tsalo** [comment]
That doesn't really fit with the naming convention of other preps. Can you expand on your workflow? Do you run QSIPrep multiple times?

**Comment by mattcieslak** [comment]
oh, that's unfortunate. Maybe the best approach would be to implement `--output-resolution auto`

**Comment by tsalo** [comment]
The HTML report is generated from scratch each time you run QSIPrep, so it _should_ include sections corresponding to the different acquisitions. If it's not doing that, then I think we have a bug.

We have talked about having `native-min` and `native-max` options for output resolution in the past.

**Comment by jhauneuro** [comment]
> oh, that's unfortunate. Maybe the best approach would be to implement `--output-resolution auto`

Yes, this would be perfect for my use case. I generally set it to the min native spatial res dimension of my data so I like the idea of `native-min` and `native-max` options!

I also have a different eddy config file for each dwi sequence to set the `mporder` parameter. I calculate that manually as per FSL guidelines (N/4 to N/2; N=number of excitations per volume), which requires knowing the number of slices and multiband factor.

**Comment by mattcieslak** [comment]
> I also have a different eddy config file for each dwi sequence to set the `mporder` parameter. I calculate that manually as per FSL guidelines (N/4 to N/2; N=number of excitations per volume), which requires knowing the number of slices and multiband factor.

This sounds like something that should also be automatically filled in based in BIDS. Are you currently doing anything else manually?

**Comment by jhauneuro** [comment]
Just those two parameters. If we can provide options to automatically set them that would be amazing. Thanks.
